src/main/java/hudson/plugins/analysis/core/BuildHistory.java

```diff
@@ -152,7 +155,7 @@ private boolean isSuccessfulAction(final ResultAction<? extends BuildResult> act
      * @return the result action
      */
     @CheckForNull
-    public ResultAction<? extends BuildResult> getResultAction(final AbstractBuild<?, ?> build) {
+    public ResultAction<? extends BuildResult> getResultAction(final Run<?, ?> build) {
```

Comment:
The diff should probably be

```
+     * @deprecated use {@link #getResultAction(AbstractBuild)}
+    */
+    @Deprecated
     public ResultAction<? extends BuildResult> getResultAction(final AbstractBuild<?, ?> build) {
+        return getResultAction((Run<?,?>)build);
+    }
+
+    @CheckForNull
+    public ResultAction<? extends BuildResult> getResultAction(final Run<?, ?> build) {
```

In order to provide full binary compatibility